web-server/src/components/TeamIncidentPRsFilter.tsx

```
@@ -0,0 +1,377 @@
+
+      <FormProvider {...addUserMethods}>
+        <FlexBox col gap2>
+          <FlexBox>
+            <Line big flexGrow={1}>
+              Include Reverted PR Incidents
+            </Line>
+            <IOSSwitch
+              checked={settings.include_revert_prs}
+              value={String(settings.include_revert_prs)}
+              onChange={(e) => {
+                const isEnabled = e.target.value === 'true' ? true : false;
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)